docs/gui/unit-info-viewer.rst

```diff
@@ -17,3 +17,9 @@ Usage
 ::
 
     gui/unit-info-viewer
+
+Overlays
+--------
+
+This tool adds progress bars, experience points and levels (affected by skill rust)
+in the unit view screen, all color-coded when enabled as a module. 
```

Comment:
being enabled as a module isn't really relevant here. also needs a description of what the colors mean